package.json

```diff
@@ -51,6 +51,8 @@
     "read-dir-files": "^0.1.1",
     "standard": "^17.0.0"
   },
+  "main": "./lib/index.js",
+  "module": "./lib/esm.mjs",
```

Comment:
```suggestion
  "main": "./lib/index.js",
```

Reply:
Alright, but then it's impossible to get the esm module in TypeScript.

Reply:
The solution for the ESM export (`import fse from fs-extra/esm`) might be to add a type declaration file, and replace the `./esm` entry in `exports` with an object that has two keys `import` and `types`.

Background on why the ESM export on `fs-extra` itself is undesired is here: https://github.com/jprichardson/node-fs-extra/pull/974#discussion_r1027022282

Reply:
PS: I'm not actually sure what the advantage of the ESM module is (in this context). Afaik you can always import the CommonJS exports, even if you're in the context of a pure ESM package.